run-desktop.py

```diff
@@ -3,5 +3,6 @@
 
 subprocess.run([
     'python',
-    'main.py'
+    'main.py',
+    *sys.argv[1:]
```

Comment:
I think it could also make sense to have a custom entrypoint script for the automated run.
This would just display the Game scene and wouldn't even need to handle (much of) pygame/user input.
It would also have the difficulty configuration from CLI.
I'll leave that for last :smile: , in the meantime I'll do as suggested :)

Reply:
Looking back, this was a quick work-around to pass the extra arguments coming from `pipenv run` into the script.

As far as I'm aware, we can't assign the first argument to a variable because it may not even exist. And I don't know how sys.args (or bash/exec) handles python's `None` in arguments (which would be the default).

An option would be to create 2 explanatory lists (with and without file argument), but I think I would prefer to go with a dedicated script/entrypoint.

Reply:
A dedicated entry point would make sense, maby something like `auto.py` instead of main. And it could take a difficulty level or custom config as an argument.

Reply:
That's the idea, to configure the difficulty, even allowing to customize the difficulty from the command line, something like:
```bash
# set base difficulty "easy" but set num_cpus to 6
pipenv run auto --easy --num-cpus 6 #...
```